### DISCUSSION TITLE
[建议]可以参考下https://win.react.yunair.cn/#/main

### DISCUSSION THREAD

**Discussion by yichen9247** [win12-online/win12#344] [General]
## 描述

可以参考下https://win.react.yunair.cn/#/main

## 方案

（请在这里提出施行建议的方案或希望有的功能）

## 其他

（如果有其他的内容，请在这里进行补充）

**Comment by NB-Group** [comment]
你的项目做的很不错！很高兴我们的项目给了你启发！
